polkadot-parachains/src/service.rs

```diff
@@ -226,10 +230,11 @@ where
 		})
 		.transpose()?;
 
-	let executor = sc_executor::NativeElseWasmExecutor::<Executor>::new(
+	let executor = sc_executor::WasmExecutor::<HostFunctions>::new(
 		config.wasm_method,
 		config.default_heap_pages,
 		config.max_runtime_instances,
+		None,
 		config.runtime_cache_size,
 	);
```

Comment:
This seems to be the most important change, the rest is just a consequence.

Reply:
Just taking out a level of indirection by always taking the same branch that was there before.

Reply:
What?